lib/rialto/etl/namespaces.rb

```diff
@@ -21,15 +22,9 @@ def remove_vocab_from_uri(vocab, uri)
       RIALTO_CONTEXT_ROLES = RDF::Vocabulary.new(rialto_base + 'context/roles/')
       RIALTO_CONTEXT_IDENTIFIERS = RDF::Vocabulary.new(rialto_base + 'context/identifiers/')
       RIALTO_CONTEXT_POSITIONS = RDF::Vocabulary.new(rialto_base + 'context/positions/')
-      SKOS = RDF::Vocabulary.new('http://www.w3.org/2004/02/skos/core#')
-      VCARD = RDF::Vocabulary.new('http://www.w3.org/2006/vcard/ns#')
-      FOAF = RDF::Vocabulary.new('http://xmlns.com/foaf/0.1/')
       VIVO = RDF::Vocabulary.new('http://vivoweb.org/ontology/core#')
-      DCTERMS = RDF::Vocabulary.new('http://purl.org/dc/terms/')
       OBO = RDF::Vocabulary.new('http://purl.obolibrary.org/obo/')
-      RDFS = RDF::Vocabulary.new('http://www.w3.org/2000/01/rdf-schema#')
-      BIBO = RDF::Vocabulary.new('http://purl.org/ontology/bibo/')
-      GEONAMES = RDF::Vocabulary.new('http://sws.geonames.org/')
+      GEONAMES = 'http://sws.geonames.org/'
```

Comment:
Why not an `RDF::Vocabulary`? Seems confusing to have an exception.

Reply:
Because there already is a GEONAMES vocabulary, with a different namespace. https://github.com/ruby-rdf/rdf-vocab/blob/develop/lib/rdf/vocab/geonames.rb#L10 and I was confused about which one we were using.